seq-library/APTw_3T_001_2uT_36SincGauss_DC90_2s_braintumor/APTw_3T_001_2uT_36SincGauss_DC90_2s_braintumor.m

```diff
@@ -1,32 +1,47 @@
 %% APTw_3T_001_2uT_36SincGauss_DC90_2s_braintumor
-% An APTw protocol with above 90% DC and tsat of 2 s:
-%
-%     pulse shape = Sinc-Gaussian
-%     B1cwpe = 2 uT
-%     n_pulses = 36
-%     tp = 50 ms
-%     td = 5 ms
-%     Tsat = n*(tp+td) = 2 s (1.975 s as last td is missing)
-%     DCsat = 0.5 and 
-%     Trec = 3.5/3.5 s (saturated/M0)
+% An APTw protocol with Sinc-Gaussian pulses,90% DC and tsat of 2 s:
```

Comment:
I'd probably put a short functionality description in front of this, just in case these files end up somewhere out of context. 
```suggestion
% Creates a sequence file for an APTw protocol with Sinc-Gaussian pulses, 90% DC and tsat of 2 s
```